packages/angular-ui-components/angular.json

```diff
@@ -27,19 +27,15 @@
 					}
 				},
 				"lint": {
-					"builder": "@angular-devkit/build-angular:tslint",
+					"builder": "@angular-eslint/builder:lint",
 					"options": {
-						"tsConfig": [
-							"./tsconfig.json",
-							"./tsconfig.spec.json"
-						],
-						"exclude": [
-							"**/node_modules/**"
-						]
+						"lintFilePatterns": ["src/**/*.ts", "src/**/*.html"]
 					}
 				}
 			}
 		}
 	},
-	"defaultProject": "@kelvininc/angular-ui-components"
```

Comment:
I think this is necessary to generate the dist in the right way

Reply:
I am getting this linting information that it is a deprecated value:

![image](https://user-images.githubusercontent.com/19409687/187470419-66db4271-c5d9-41dc-878e-56c132213790.png)

Reply:
🤔 
> defaultProject workspace option has been deprecated (the default is now the current working directory

Let's see whats happen